Makefile

```diff
@@ -25,6 +25,9 @@ bisect: clean test
 finished-bisect: clean finished-test
 	bisect-ppx-report html
 
+play:
+	$(OCAMLBUILD) -tag 'debug' $(MAIN) && OCAMLRUNPARAM=b ./$(MAIN)
+	
```

Comment:
Did you get this running? If not, it is something that should be kept in mind to fix later (not prevalent for the demo, but prevalent in the long run)

Reply:
Not on my computer, so I sent the code to Tise to try on her end. I think it is an error with my environment rather than the code. I am planning to ask for more help on office &/or consulting hours when I get time.